.github/workflows/continuous-integration.yaml

```diff
@@ -36,3 +27,31 @@ jobs:
       - name: contract - test
         run: npm run test-ci
         working-directory: ./contract
+
+  prettier:
+    runs-on: ubuntu-latest
+    needs: tests
+    steps:
+    - uses: actions/checkout@v2
+
+    # run prettier, and also ensure any lock file changes are reverted
+    - name: Prettify web-client
+      run: yarn && yarn prettier && git restore yarn.lock
+      working-directory: ./web-client
+
+    # run prettier, and also ensure any lock file changes are reverted
+    - name: Prettify web-server
+      run: yarn && yarn prettier && git restore yarn.lock && git restore package-lock.json
```

Comment:
Any `package-lock.json` should basically just be deleted from the repo as it only exists when someone accidentally use `npm install` 😄

Reply:
I removed the one package-lock.json and updated things to be "yarn" everywhere :-)